CONTRIBUTING.md

```diff
@@ -54,7 +54,9 @@ Eg. Run a script when a user says ____
 - Line 2 must always be `activation_example:` followed by a short description of how this parser would be activate
 - Line 3 must always be `regex:` followed by a regex expression that validates if the parser should run. Do not include the opening and closing `/`
 - Line 4 must always be `flags:` followed by the regex flags needed for line 3. If no flags are needed then leave the line as `flags:`
-- Line 5 must always be `*/`
+- *Optional*: `order:` that the parser should run (higher orders run first, default value is `100`).
```

Comment:
This is opposite of how order behavior usually works in ServiceNow. We really should do the opposite